docker/simnet/bitcoin-lightning/bitcoin-lightning.simnet.primary.conf

```diff
@@ -0,0 +1,23 @@
+[Application Options]
+
+maxpendingchannels=10
```

Comment:
On lnd repo there is config with every config line explained, I believe we should need to put those comments there.